api/_mixins/DocumentOrShadowRoot__Document.json

```diff
@@ -396,6 +396,288 @@
           }
         }
       },
+      "getAnimations": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/getAnimations",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "84"
+              },
+              {
+                "version_added": "83",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#enable-experimental-web-platform-features"
+                  }
+                ]
+              },
+              {
+                "version_added": "67",
+                "version_removed": "83",
+                "partial_implementation": true,
+                "notes": "Implements an older version of the specification and not supported on <code>ShadowRoot</code>, see <a href='https://crbug.com/828585'>Chromium bug 828585</a>.",
```

Comment:
The `ShadowRoot` parts of the notes can be simplified away.

If there was a way to just delete all of this flag data I would be very happy to see that too, really, but only the oldest entry is older than 2 years :/